### PR TITLE
[Docs] Count FLOPs and parameters

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/common_usage/model_analysis.md [hunk 1]
```diff
@@ -0,0 +1,740 @@
...
+
+  ```python
+  print(analysis_results['out_table'])
+  ```
+
+  <details>
+    <summary>点击展开</summary>
+
+  ```
+    +------------------------+----------------------+------------+--------------+
```
Comment (HAOCHENYE) [inline docs/zh_cn/common_usage/model_analysis.md:45]: These are some strange indents on the table. Is this expected?

Comment (zhouzaida) [reply docs/zh_cn/common_usage/model_analysis.md]: done

**Review by C1rN09 — DISMISSED** [review]

File: docs/zh_cn/common_usage/model_analysis.md [hunk 2]
```diff
@@ -0,0 +1,740 @@
+# 统计模型计算量和参数量
+
+1. 定义模型
+
+   ```python
+   import torch.nn.functional as F
+   import torchvision
+   from mmengine.model import BaseModel
+   from mmengine.analysis import get_model_complexity_info
```
Comment (C1rN09) [inline docs/zh_cn/common_usage/model_analysis.md:9]: 感觉这一行移到 2. 的代码段里面更好

File: docs/zh_cn/common_usage/model_analysis.md [hunk 3]
```diff
@@ -0,0 +1,740 @@
...
+  ```python
+  print(analysis_results['out_table'])
+  ```
+
+  <details>
+    <summary>点击展开</summary>
+
+  ```
+  +------------------------+----------------------+------------+--------------+
+  | module                 | #parameters or shape | #flops     | #activations |
```
[the remark this answers is not in the document]
Comment (zhouzaida) [reply docs/zh_cn/common_usage/model_analysis.md:46]: 加个 html 渲染就会正常了，我加一下

File: docs/zh_cn/common_usage/model_analysis.md (hunk 2 above)
Comment (zhouzaida) [reply docs/zh_cn/common_usage/model_analysis.md:9]: OK

**Pull request merged**